02c_layout.ipynb

```diff
@@ -0,0 +1,752 @@
+    "target=\"_blank\">GISS Surface Temperature Analysis (NASA)</a>\n",
+    "</p><p>\n",
+    "This site is based on data downloaded from the following site on 2024-06-17:\n",
+    "<a href=\"https://data.giss.nasa.gov/gistemp/graphs/graph_data/Global_Mean_Estimates_based_on_Land_and_Ocean_Data/graph.csv\"\n",
+    "target=\"_blank\">Global Annual Mean Surface Air Temperature Change (NASA)</a>\n",
+    "'''"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e9d46022-ef0a-4d0b-bb46-033cd9f8330b",
+   "metadata": {},
+   "source": [
+    "### Layout widget objects\n",
+    "\n",
+    "In most cases, we can use the `Layout` object to set the layout of a particular widget on initialization.  Let's display the HTML variables we just defined in `HTML` widgets using the `Layout` object to set the width of the widgets to be the same."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "187d1f8a-7514-4c09-b019-233b7829acef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| export\n",
+    "intro_text = widgets.HTML(value = INTRO_TEXT, layout = widgets.Layout(max_width = '500px'))\n",
+    "data_source_text = widgets.HTML(value = SOURCES_TEXT, layout = widgets.Layout(max_width = '500px'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4c89d351-c9a5-44df-8c12-3669f5c047df",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "intro_text"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0395d883-2a40-4b0f-83b1-b634869cd59c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_source_text"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a2ddda7-6828-492a-ba97-59b1110227a1",
+   "metadata": {},
+   "source": [
+    "### Arranging Widgets with HBox and VBox Containers\n",
+    "\n",
+    "The arrangement where every widget is stacked one on top of the other isn't ideal for a data dashboard that we expect users to access from a desktop. Jupyter widgets has several container widgets to arrange widgets in various ways. Perhaps two of the most handy are the `HBox` and `VBox` widgets, which arrange widgets horizontally and vertically, respectively. If you are familiar with `FlexBox`, [those properties are available under the hood](https://ipywidgets.readthedocs.io/en/7.6.3/examples/Widget%20Styling.html#The-Flexbox-layout), but we will not cover them here (we will use one of those properties below). \n",
+    "\n",
+    "Let's create a `HBox` to put the curve parameter widgets side by side."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9076da82-ae76-4153-9834-848209206c59",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "window_size"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b0f5d9a8-5f28-4f76-816e-9f6bd65fe1f5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "poly_order"
```

Comment:
Why are `window_size` and `poly_order` displayed again here? Seems like making the `HBox` is the next step....